pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/ClinicalKitDao.java

```diff
@@ -29,8 +33,24 @@ public class ClinicalKitDao {
                     + "LEFT JOIN ddp_participant as p on (p.participant_id = inst.participant_id) "
                     + "LEFT JOIN ddp_instance as ddp on (ddp.ddp_instance_id = p.ddp_instance_id) "
                     + "LEFT JOIN sm_id_type sit on (sit.sm_id_type_id = sm.sm_id_type_id) "
-                    + "LEFT JOIN kit_type ktype on ( sit.kit_type_id = ktype.kit_type_id) WHERE sm.sm_id_value = ? "
-                    + "AND NOT sm.deleted <=> 1 ";
+                    + "LEFT JOIN kit_type ktype on ( sit.kit_type_id = ktype.kit_type_id) "
```

Comment:
Looks like the value of `SQL_GET_CLINICAL_KIT_BASED_ON_SM_ID_VALUE` has not actually changed here.  If that's the case but we wanted to change the whitespace, please put in a PR comment to that effect so reviewers know--or leave the whitespace as is.

Reply:
only whitespace change because I like having the `WHERE` start in a new line for easier reading of query